packages/static-build/test/build-fixtures/14-gatsby-with-esm-config/vercel.json

```diff
@@ -0,0 +1,13 @@
+{
+  "version": 2,
+  "builds": [
+    {
+      "src": "package.json",
+      "use": "@vercel/static-build",
+      "config": {
+        "zeroConfig": true,
+        "framework": "gatsby"
```

Comment:
I believe vercel.json can be simplified to a single line now that we support framework

https://vercel.com/docs/project-configuration#project-configuration/framework

```json
{
  "framework": "gatsby"
}
```

Reply:
<picture data-single-emoji=":true-dat:" title=":true-dat:"><img class="emoji" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..tHETQvT_fgFpRfL4.ezq_66GMRdhaR8S8K6tyz2lxbanWSvrjZRixPyQ4YHCiYVinT6VR3PSzsh9TWAGQQPcOCstZkhkjHt64Ij0IEzrl_5kd_tOax7JIPxrYInp_o3M3p0qU2yqc.CGWglEVWjTF1gBA2ZamO6w" alt=":true-dat:" width="20" height="auto" align="absmiddle"></picture> I'm going to merge as is since this copies the other ones and I don't want to gold plate in this PR. Going to be modifying static-build including tests soon so I'll make sure to incorporate this recommended change then